### ISSUE TITLE
Support 'artifactrepo: [url];layout=p2' setting. 

### ISSUE THREAD

**Issue by jwausle** [ToToTec/cmvn#3]
Support '`artifactrepo: [url];id=inplace;layout=p2`' setting 
to generate pom like:

```
    <repository>
      <id>target</id>
      <layout>p2</layout>
      <releases>
        <enabled>true</enabled>
      </releases>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
      <url>[url]</url>
    </repository>
```

Otherwise:

```
==== cmvn with alias, use --alias-help to display aliases ====
Exception in thread "main" java.lang.RuntimeException: Error while reading file: .../cmvn.conf
    at de.tototec.cmvn.ProjectReaderImpl.readConfigFile(ProjectReaderImpl.java:60)
    at de.tototec.cmvn.UnconfiguredCmvnProject.allSubProjects$lzycompute(UnconfiguredCmvnProject.scala:79)
    at de.tototec.cmvn.UnconfiguredCmvnProject.allSubProjects(UnconfiguredCmvnProject.scala:78)
    at de.tototec.cmvn.UnconfiguredCmvnProject.configureProjectRecursive(UnconfiguredCmvnProject.scala:94)
    at de.tototec.cmvn.CmvnApp2$.main(CmvnApp2.scala:88)
    at de.tototec.cmvn.CmvnApp2.main(CmvnApp2.scala)
Caused by: java.lang.RuntimeException: Unsupported repository option: KeyValue(layout,p2,/Users/winter/Git/bmw-root/SWA_Prototype.2/core/tests/com.bmw.aida.cheetah.tests.cheetah10/../../../.cmvn/cmvn-repos.conf,1)
    at de.tototec.cmvn.CmvnConfigKey$6.read(CmvnConfigKey.java:136)
    at de.tototec.cmvn.ProjectReaderImpl.readConfigFile(ProjectReaderImpl.java:52)
```

**Comment by lefou** [comment]
Implemented by #4.

Needs documentation though.

**Comment by lefou** [comment]
Done
